bluebottle/clients/context.py

```diff
@@ -0,0 +1,12 @@
+from django.template import Context
+from bluebottle.clients import properties
+
+class ClientContext(Context):
+    """
+    A Context with builtin properties support. Mostly meant
+    for rendering mail templates that don't get the
+    ContextProcessor goodness
+    """
+    def __init__(self, *args, **kw):
+        super(ClientContext, self).__init__(*args, **kw)
+        self.update({'tenant_properties':properties})
```

Comment:
Should we pass the whole context to the template? This also includes secret keys etc.